src/styles/modal.scss

```diff
@@ -110,8 +110,10 @@ $modalForegroundZIndex: 12000;
           color: inherit;
           text-decoration: none;
           opacity: 0.75;
-          &:hover {
+          &:hover,
+          &:focus {
             text-decoration: underline;
+            outline: none;
```

Comment:
I'm not sure about the `outline: none`, which is why I didn't add it anywhere. I find it too opinionated. Right now, Uppload links, buttons, etc., would just inherit the `:focus` states of the webpage they're running on, and maybe people want the default outlines? Would love to hear thoughts.

Reply:
I added the same styling to `:focus` as is currently on `:hover`. Thus seeing the default outline is no longer necessary right?